diff --git a/config/compiler/BUILD.gn b/config/compiler/BUILD.gn
index 94fc4b4..3badbb9 100644
--- a/config/compiler/BUILD.gn
+++ b/config/compiler/BUILD.gn
@@ -1499,7 +1499,7 @@ config("compiler_cpu_abi") {
           cflags += [ "-maix64" ]
           ldflags += [ "-maix64" ]
         }
-      } else if (is_clang) {
+      } else if (is_clang && false) {
         cflags += [ "--target=powerpc64le-unknown-linux-gnu" ]
         ldflags += [ "--target=powerpc64le-unknown-linux-gnu" ]
       } else {
@@ -1507,13 +1507,13 @@ config("compiler_cpu_abi") {
         ldflags += [ "-m64" ]
       }
     } else if (current_cpu == "riscv64") {
-      if (is_clang && !is_android) {
+      if (is_clang  && false && !is_android) {
         cflags += [ "--target=riscv64-linux-gnu" ]
         ldflags += [ "--target=riscv64-linux-gnu" ]
       }
       cflags += [ "-mabi=lp64d" ]
     } else if (current_cpu == "loong64") {
-      if (is_clang) {
+      if (is_clang && false) {
         cflags += [ "--target=loongarch64-linux-gnu" ]
         ldflags += [ "--target=loongarch64-linux-gnu" ]
       }
@@ -1522,7 +1522,7 @@ config("compiler_cpu_abi") {
         "-mcmodel=medium",
       ]
     } else if (current_cpu == "s390x") {
-      if (is_clang) {
+      if (is_clang && false) {
         cflags += [ "--target=s390x-unknown-linux-gnu" ]
         ldflags += [ "--target=s390x-unknown-linux-gnu" ]
       }
